crates/turbopack-css/src/process.rs

```diff
@@ -269,6 +275,8 @@ pub enum ParseCssResult {
         #[turbo_tasks(debug_ignore, trace_ignore)]
         cm: Arc<swc_core::common::SourceMap>,
 
+        code: String,
```

Comment:
We already have the source code lying around somewhere in a Vc. It seems unnecessary to clone that into this and store it again